### ISSUE TITLE
How to get the tfrecord document?

### ISSUE THREAD

**Issue by PFpengfeng** [valeriano-manassero/tfx-kubeflow-pipelines#21]
Hi, Sir.  I am trying to run your pipeline [cifar-10 interactive.ipynb](https://github.com/valeriano-manassero/tfx-kubeflow-pipelines/tree/master/cifar-10). But using my tfrecord documents, the code can't work. I transform img into bytes form and label into int64 form.  So I want to ask how to get the tfrecord document. Please help me. Thank you!

**Comment by valeriano-manassero** [comment]
Hi,
not sure if I got what you need but I used code similar to this to create tfrecords in that case:
```
image_resized = tf.cast(tf.image.resize_images(image, size=[IMG_SIZE,IMG_SIZE], method=tf.image.ResizeMethod.AREA), tf.uint8)
image_string = tf.image.encode_jpeg(image_resized)
feature = {
    'labels': tf.train.Feature(bytes_list=tf.train.BytesList(
        value=[tf.compat.as_bytes(labels)])),
   'image_raw': tf.train.Feature(bytes_list=tf.train.BytesList(
        value=[image_string.numpy()])),
}
tf_example = tf.train.Example(features=tf.train.Features(feature=feature))
writer.write(tf_example.SerializeToString())
```
this is the info you need?
If not feel free to ask more!

**Comment by PFpengfeng** [comment]
Thank you very much! I've run the code successfully.
